### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by NathanBnm**
Note: I think that "Caesae Shift Cipher", "Atbash Cipher"... should be added in translations too (from the left-side panel).

Maybe you should adapt Wikipédia links depending of the language.

### CODE REVIEW

**Comment by arshubham** [comment]
@NathanBnm All issues except the wiki link relating to translations have been resolved. Give me a few days, I am working on a redesign. Therefore, Some strings might be added or removed.

**Comment by NathanBnm** [comment]
> @NathanBnm All issues except the wiki link relating to translations have been resolved. Give me a few days, I am working on a redesign. Therefore, Some strings might be added or removed.

Okay I will update them when the new app will be released.

**Comment by arshubham** [comment]
@NathanBnm The redesign has been merged. You can update the translations now. Please do tell me if I missed any string.

**Comment by NathanBnm** [comment]
@arshubham I just fixed conflicts. Translations are now correct and you can merge this PR

File: po/fr.po [hunk 1]
```diff
@@ -1,177 +1,178 @@
...
 #: src/views/CipherView.vala:82
 #: data/com.github.arshubham.cipher.appdata.xml.in:58
 msgid "ASCII Encoding"
-msgstr ""
+msgstr "Encodage ASCII"
 
 #: src/views/CaesarCipherView.vala:61
 msgid "Number of letters to shift to the right: "
-msgstr ""
+msgstr "Nombre de lettres à décaler vers la droite : "
```
Comment (arshubham) [inline po/fr.po]: Please remove the extra space.
```suggestion
msgstr "Nombre de lettres à décaler vers la droite: "
```

File: po/fr.po [hunk 2]
```diff
@@ -1,177 +1,178 @@
...
 #: data/com.github.arshubham.cipher.appdata.xml.in:55
 msgid "Caesar Shift Cipher"
-msgstr ""
+msgstr "Chiffre de César"
 
 #: src/views/CipherView.vala:77
 #: data/com.github.arshubham.cipher.appdata.xml.in:56
 msgid "Atbash Cipher"
-msgstr ""
+msgstr "Atbash"
```
Comment (arshubham) [inline po/fr.po]: Can All the Cipher names be started with "Chiffre de" tfor consistancy as in Chiffre de César?

File: po/fr.po [hunk 3]
```diff
@@ -1,177 +1,178 @@
...
 
 #: src/widgets/HeaderBar.vala:91
 msgid "Dark background"
-msgstr ""
+msgstr "Thème sombre"
 
 #: src/widgets/HeaderBar.vala:131
 msgid "Click to learn more about "
-msgstr ""
+msgstr "Cliquez pour en savoir plus"
```
Comment (arshubham) [inline po/fr.po]: Please add extra space here as cipher name will be inserted after this.
```suggestion
msgstr "Cliquez pour en savoir plus "
```

File: po/fr.po (hunk 1 above)
Comment (NathanBnm) [reply po/fr.po]: In French the colon must be preceded by a space +1
For further information you can read this article. French punctuation has special rules : https://www.iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation

File: po/fr.po (hunk 2 above)
Comment (NathanBnm) [reply po/fr.po]: I checked on Wikipédia, and in the French version it seems that "chiffre" is not mandatory for these names.

File: po/fr.po (hunk 3 above)
Comment (NathanBnm) [reply po/fr.po]: Ah okay I am going to change that because in this case it should be:
msgstr "Cliquez pour en savoir plus sur "

**Comment by arshubham** [comment]
@NathanBnm Please resolve the conflict. There was a string change due to @ryonakano 's contribution.

**Comment by NathanBnm** [comment]
@arshubham fixed :+1:

**Pull request merged**